packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -114,10 +119,33 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
             this.updateXY();
         }
         if (this.lineChartRef.current) {
+            if (this.preventDefaultHandler === undefined) {
+                this.preventDefaultHandler = (event: WheelEvent) => {
+                    if (event.ctrlKey) {
+                        event.preventDefault();
+                    }
+                };
+                const xyMainElement = document.getElementById(this.getUniqueDivId());
+                if (xyMainElement) {
+                    xyMainElement.addEventListener('wheel', this.preventDefaultHandler);
+                } else {
+                    this.preventDefaultHandler = undefined;
+                }
+            }
             this.lineChartRef.current.chartInstance.render();
         }
     }
 
+    componentWillUnmount(): void {
+        if (this.preventDefaultHandler !== undefined) {
+            const xyMainElement = document.getElementById(this.getUniqueDivId());
+            if (xyMainElement) {
+                xyMainElement.removeEventListener('wheel', this.preventDefaultHandler);
+            }
+            this.preventDefaultHandler = undefined;
+        }
+    }
+
```

Comment:
The timegraph is able to prevent the mouse wheel document zoom without resorting to adding/removing the handler and needing uniqueDivId. I think it would be worth to try and simplify the code by using a similar strategy.

Reply:
Patrick suggested using `React.createRef()` for this, and I agree. It could be implemented by:

1. Adding `private chartRef: any;` below line 41;

2. Adding `this.chartRef = React.createRef();` below line 91;

3. Replacing most of the code block above with `this.chartRef.current.addEventListener('wheel', this.preventDefaultHandler);`. No need for `xyMainElement`.

Also, in this case it seems that the eventListener is being cleared by the garbage collector. No need to remove it manually.